test/services/PageBuilderSpec.scala

```diff
@@ -618,14 +621,148 @@ class PageBuilderSpec extends BaseSpec with ProcessJson with StanzaHelper {
           // Test contents of individual pages
           testSqpQp(indexedSeqOfPages(0))
 
-          testSqpFap(indexedSeqOfPages(1))
+          testSqpSap(indexedSeqOfPages(1))
+
+          testSqpFap(indexedSeqOfPages(2))
 
-          testSqpSap(indexedSeqOfPages(2))
 
         case Left(err) => //fail(s"Flow error $err")
       }
     }
 
+    // "Return ungrouped Instruction stanzas when text contents do not start with similar text" in {
```

Comment:
Can we remove this commented out code?

Reply:
These are bullet point tests which should be moved out of the PageBuilderSpec class in the viewer code, but until this is done, its easier to do the comparison between the viewer code and backend code when syncing. I've been leaving the move to Richard as he fully understands the bullet point stuff